### PR TITLE
Makefile: Add default task

### PR DESCRIPTION
**Pull request by shenli**
@BusyJay @coocood @siddontang @qiuyesuifeng PTAL

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -37,9 +37,11 @@ LDFLAGS += -X "github.com/pingcap/tidb/util/printer.TiDBGitHash=$(shell git rev-
 
 TARGET = ""
 
-.PHONY: all build install update parser clean todo test gotest interpreter server goyacc golex dev
+.PHONY: default all build install update parser clean todo test gotest interpreter server goyacc golex dev
```
Comment (siddontang) [inline Makefile]: need default here?

Comment (shenli) [inline Makefile]: PTAL @siddontang

**Comment by siddontang** [comment]
LGTM

**Comment by coocood** [comment]
LGTM

**Pull request merged**